http.go

```diff
@@ -301,3 +333,53 @@ func (h *httpGetter) Remove(ctx context.Context, in *pb.GetRequest) error {
 	}
 	return nil
 }
+
+func DefaultServerErrorHandler(ctx context.Context, w http.ResponseWriter, r *http.Request, err error) {
```

Comment:
Should probably be unexported?

Reply:
I had it exported so that you can use it as a fallback in your own handlers, in case we add more error types.